CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v0.73.0
```

Comment:
Where's v0.72?  Not sure if the versions are out of sync a bit…

Reply:
0.72.0 was released ~hour ago :)

Reply:
🤦‍♂️